domains/blockbreaker.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheBlockbreaker",
+        "email": "burstsgodly@gmail.com"
+    },
+    "record": {
+        "TXT": "did=did:plc:agavl34uosnej4h54haas6aj"
```

Comment:
TXT record alone doesn't do anything.

Reply:
I don’t remember trying to re register that domain with a txt record….